loadtest/README.md

```diff
@@ -0,0 +1,57 @@
+Load Test for Khan API
+======================
+
+This application performs a specified amount of sequential requests to a remote Khan API server, with a specified time period between two consecutive requests. Usage: `../khan loadtest --help`
```

Comment:
we're not sending concurrent requests? or will it be achieved by running several pods of the load test instance?

Reply:
The latter option you mentioned, multiple pods knowing the shared clans.

Reply:
It's not a huge deal, since your proposal works. But it would be nice to be able to run concurrently in a local environment. Maybe add a concurrency parameter and instantiate several go routines?

Reply:
Sounds nice! I'll do that!